crates/sui/src/client_commands.rs

```diff
@@ -609,6 +609,18 @@ pub enum SuiClientCommands {
         address_override: Option<ObjectID>,
     },
 
+    /// Profile the gas usage of a transaction. Outputs a file `gas_profile_{tx_digest}_{unix_timestamp}.json` which can be opened in a flamegraph tool such as speedscope.
+    #[clap(name = "profile-transaction")]
+    ProfileTransaction {
+        /// The digest of the transaction to replay
+        #[arg(long, short)]
+        tx_digest: String,
+
+        /// If specified, overrides the filepath of the output profile, for example -- /temp/my_profile.json
+        #[clap(name = "profile_output_filepath")]
+        profile_output_filepath: Option<PathBuf>,
```

Comment:
Similar comment here on short command-line options.

```suggestion
        profile_output: Option<PathBuf>,
```